docs/Next-Step-Server-REST-API-Reference.md

```diff
@@ -7973,6 +7973,7 @@ The list of expected status codes:
     "credentialName": "RETAIL_CREDENTIAL",
     "credentialType": "PERMANENT",
     "username": "username1234",
+    "credentialValue": "JeM1vr%GyJFh",
```

Comment:
Isn't missing also timestampExpires property from UpdateCredentialRequest?

Reply:
Yes, it is. However, we should describe in the future which parameter is optional and which is mandatory - for the entire API documentation.